phases/ephemeral/witx/wasi_ephemeral_profile.witx

```diff
@@ -0,0 +1,27 @@
+(use "wasi_ephemeral_args.witx")
+(use "wasi_ephemeral_clock.witx")
+(use "wasi_ephemeral_environ.witx")
+(use "wasi_ephemeral_fd.witx")
+(use "wasi_ephemeral_path.witx")
+(use "wasi_ephemeral_poll.witx")
+(use "wasi_ephemeral_proc.witx")
+(use "wasi_ephemeral_random.witx")
+(use "wasi_ephemeral_sched.witx")
+(use "wasi_ephemeral_sock.witx")
+
+;;; The WASI standard specifies executables as having all of the
+;;; WASI modules available for import, and exporting a start function.
+(profile $wasi_ephemeral_executable
+  ;;; Executables must export a start function:
+  (@interface func (export "_start"))
+  (import $wasi_ephemeral_args)
+  (import $wasi_ephemeral_clock)
+  (import $wasi_ephemeral_environ)
+  (import $wasi_ephemeral_fd)
+  (import $wasi_ephemeral_path)
+  (import $wasi_ephemeral_poll)
+  (import $wasi_ephemeral_proc)
+  (import $wasi_ephemeral_random)
+  (import $wasi_ephemeral_sched)
+  (import $wasi_ephemeral_sock)
```

Comment:
With [module imports](https://github.com/WebAssembly/module-types/) on their way, this syntax has the risk of being confusing. Would it make sense to use a different keyword instead of `import` here, like `(include ...)` or `(import ...)`?

Reply:
`(expose ...)` ?

Reply:
`expose` works for me.

Reply:
Implemented in https://github.com/WebAssembly/WASI/pull/267/commits/77229818a40dae9be3335731d045fcd59e82ab80